community/recruitment.md

```diff
@@ -11,10 +11,10 @@ We recruit UG 1st years in the team, preferably after they have spent one semest
 For more detailed description, refer [Freshers' Selection](/docs/events/freshers-selection)
 
 ## Do we accept M.Tech and Ph.D students?
-By acceptance, we mean that they will hold a number of responsibilities
+No. By acceptance, we mean that they will hold a number of responsibilities
 - Attend our meetings
 - Participate in our workshops and their logistics
 - Collaborate on projects together
 - Be a great team member
```

Comment:
We need to remove/improve this line, it doesn't specify anything and is _hand wavy_. What is a **Great team member**?

Reply:
We can remove the last line - `Be a Great team member`
or replace with something like - `Be an active member of our community` 

What do you suggest?

Reply:
Active member of the society sounds bettter

Reply:
Modified to `Be an active member of the society`